main.go

```diff
@@ -55,6 +58,39 @@ Flags:
 
 `
 
+func outputDot(fname string, outputFormat string) {
+	// get cmdline default for analysis
+	opts := analysisSetup()
+	if e := processListArgs(&opts); e != nil {
+		log.Fatalf("%v\n", e)
+	}
+
+	output, err := Analysis.render(opts)
+	if err != nil {
+		log.Fatalf("%v\n", err)
+	}
+
+	log.Println("writing dot output..")
+	writeErr := ioutil.WriteFile(fmt.Sprintf("%s.gv", fname), output, 0755)
+	if writeErr != nil {
+		log.Fatalf("%v\n", writeErr)
+	}
+
+	if outputFormat == "svg" {
+		log.Println("converting dot to svg..")
```

Comment:
It seems unnecessary to have these if cases here. You could just have single one that uses the `outputFormat` variable.